packages/nimble-components/src/label-provider/rich-text-editor/label-tokens.ts

```diff
@@ -0,0 +1,26 @@
+import { DesignToken } from '@microsoft/fast-foundation';
+import { richTextEditorLabelDefaults } from './label-token-defaults';
+
```

Comment:
Looking at the [localization token naming rules](https://github.com/ni/nimble/blob/d3969e869fd98721c65126c58543c0de3c9c1100/packages/nimble-components/CONTRIBUTING.md#localization)

I'd say these are 
element/types: rich-text
part/category: optional and omit, don't think these are ambiguous about what they are associated with or likely to conflict with future changes in rich text
functionality/sub-part: toggle-bold, toggle-italic, etc

So I'd vote for the names:
```
rich-text-toggle-bold-label
rich-text-toggle-italics-label
rich-text-toggle-bulleted-list-label
rich-text-toggle-numbered-list-label
```

@jattasNI @msmithNI does that sound reasonable?

Reply:
Updated as per your suggestion